### DISCUSSION TITLE
Is it a calculation bug that non-empty directory have 0 size?

### DISCUSSION THREAD

**Discussion by PegasusPlusUS** [nushell/nushell#14362] [Q&A] [answered]
Please see attached screen shot, VMM size is 0:

![Uploading Screenshot 2024-11-16 201832.png…]()

**Comment by PegasusPlusUS** [comment]
![Screenshot 2024-11-16 201832](https://github.com/user-attachments/assets/356d39c4-a789-431e-b616-ae4944d9397a)

**Reply by fdncred** [reply]
Nushell just prints what the file system returns. There is no calculation when doing `ls` by itself.

**Reply by PegasusPlusUS** [reply]
I compare result of PowerShell and bash, they both give every directory of size 0 or blank, while nushell gives mostly 4KB, several 8KB, and this VMM of 0, it's quite confusing, :(

**Reply by fdncred** [reply, marked as answer]
Neither powershell nor bash are written in rust, but this seems to be pretty common. I've seen many articles like this one that explain 4kb folders. https://askubuntu.com/questions/186813/why-does-every-directory-have-a-size-4096-bytes-4-k. I have no idea why VMM is 0 but I suspect it's a file system thing.
